src/networks.ts

```diff
@@ -16,6 +14,16 @@ const NETWORK_CONFIG = {
   polygon: POLYGON_URL,
 } as const
 
+type NetworkName = keyof typeof NETWORK_CONFIG
+
+/**
+ * Configuration options for initializing the SubsquidClient
+ */
+type SubsquidClientOptions = {
+  customSubsquidUrl?: string;
+  network?: string;
+}
+
 const SUPPORTED_NETWORKS = Object.keys(NETWORK_CONFIG) as NetworkName[]
 
-export { NETWORK_CONFIG, SUPPORTED_NETWORKS, type NetworkName }
+export { NETWORK_CONFIG, SUPPORTED_NETWORKS, type NetworkName, type SubsquidClientOptions }
```

Comment:
why not exporting all inline and save a line?

Reply:
wdym ? like `*` ? 

```ts
export { NETWORK_CONFIG, SUPPORTED_NETWORKS } ...
export type { NetworkName, type SubsquidClientOptions } ...
```

is an alternative I'd make

Reply:
i mean more like `export const SUPPORTED_NETWORKS = Object.keys(NETWORK_CONFIG) as NetworkName[]
`

Reply:
gotcha, rather not allocate a new array just for _syntax sugar_, leaning for this then 👍🏼

Reply:
LOL